### PR TITLE
Feat: polyfill CSS `placeContent` support on native

### PR DESCRIPTION
**Pull request by ecreeth**
### Summary
This PR adds the polyfill for [`placeContent`](https://developer.mozilla.org/en-US/docs/Web/CSS/place-content)

### Test plan

EDIT:  We only support single values that are common to both properties (alignContent, justifyContent)

### CODE REVIEW

**Comment by necolas** [comment]
Thanks. We should only support single values that map the same to both long form properties, except for first/last baseline which is only for alignItems IIRC. This is generally how stylex and React Native work for other longforms

**Comment by ecreeth** [comment]
I've updated the PR. Now only single values are supported.

File: packages/react-strict-dom/tests/css-test.native.js [hunk 1]
```diff
@@ -1532,3 +1532,73 @@ describe('queries: @media', () => {
...
+      case: { placeContent: 'flex-start center' }
+    });
+    expect(css.props.call(mockOptions, styles.case)).toMatchSnapshot();
+
+    expect(console.warn).toHaveBeenCalledWith(
+      expect.stringContaining(
+        'unsupported "placeContent" style property value "flex-start center"'
+      )
+    );
+  });
```
Comment (ecreeth) [inline packages/react-strict-dom/tests/css-test.native.js:1603]: Maybe we should show a different message when multiple values are used.

**Comment by necolas** [comment]
Thanks! I made a few changes and merged this into main

**Pull request closed without merging**